pkg/dbmate/db.go

```diff
@@ -148,25 +153,41 @@ func (db *DB) Wait() error {
 	return db.wait(drv)
 }
 
-// CreateAndMigrate creates the database (if necessary) and runs migrations
+// CreateAndMigrate creates the database (if necessary), load schema.sql file
+// to the database (if autoload enabled) and then apply all pending migrations
 func (db *DB) CreateAndMigrate() error {
```

Comment:
If we constantly update the `schema.sql`, then we don't need to apply migrations after the import